zboss/Kconfig.sources

```diff
@@ -0,0 +1,313 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+# This file defines options, that are used to set configurable entries of zb_vendor.h and zb_config.h platform files.
+# It will be used only if ZBOSS source code is available.
+
+menu "ZBOSS features"
+	visible if ZBOSS_LIBRARY_SOURCES
```

Comment:
This only remove visibility in limited view in Kconfig.
If user toggles `show-all` in menuconfig, then those settings can still be viewed.

Reply:
The reasoning behind this is explained in: https://github.com/NordicPlayground/nrfxlib/pull/166#discussion_r420679677

Reply:
Changed to regular `if` in NCS platform repository.